Pipfile

```diff
@@ -18,7 +18,7 @@ xworkflows = "*"
 "fhir.resources" = "*"
 httpx = "*"
 slowapi = "*"
-uvicorn = {extras = ["standard"], version = "*"}
+uvicorn = "==0.14.0"
```

Comment:
This is great. 
We should also start giving thought to pinning our other dependencies as well as we standardize in the next iteration.